cmd/lncli/invoicesrpc_active.go

```diff
@@ -214,9 +218,18 @@ func addHoldInvoice(ctx *cli.Context) error {
 
 	switch {
 	case ctx.IsSet("amt"):
-		amt = ctx.Int64("amt")
+		if ctx.IsSet("amt_msat") {
+			return fmt.Errorf("amt and amt_msat are mutually exclusive")
+		}
+		ms = ctx.Int64("amt") * 1000
+	case ctx.IsSet("amt_msat"):
+		if ctx.IsSet("amt") {
+			return fmt.Errorf("amt and amt_msat are mutually exclusive")
+		}
+		ms = ctx.Int64("amt_msat")
 	case args.Present():
-		amt, err = strconv.ParseInt(args.First(), 10, 64)
+		amt, err := strconv.ParseInt(args.First(), 10, 64)
+		ms = amt * 1000
```

Comment:
Nit: move this below the error check. You can also use `lnwire.NewMSatFromSatoshis` instead of the multiplier.

Reply:
Thanks. Now as I'm passing both amt and amtMsat to RPC as is, no conversion is needed.